### ISSUE TITLE
Editorial change to 3.4.2 Example

### ISSUE THREAD

**Issue by johnfoliot** [w3c/personalization-semantics#44]
The current code example shows the following:
     <input aui-simplification="critical" aria-function="submit" value="Submit" type="submit"/>

However, the current list of allowable token values is simplets, medium, or low (there is no "critical").

Additionally, there is currently no such attribute known as "aria-function" - this should be removed from the example. (source: https://www.w3.org/TR/wai-aria-1.1/)

**Comment by clapierre** [comment]
Closed via #86
